### PR TITLE
sync-integration-tests-fix-paths

### PR DESCRIPTION
**Pull request by isabelrios**
This PR fixes some paths so that these tests can run anywhere without having to modify them. 

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py [hunk 1]
```diff
@@ -58,12 +59,22 @@ def tps_addon(pytestconfig, tmpdir_factory):
 def tps_config(fxa_account, monkeypatch):
     monkeypatch.setenv('FXA_EMAIL', fxa_account.email)
     monkeypatch.setenv('FXA_PASSWORD', fxa_account.password)
-    with open ("/Users/synctesting/.jenkins/workspace/fenix/app/src/androidTest/resources/email.txt", "w") as f:
+
+    os.chdir('../../../../..')
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py]: probably for readability we should use the absolute paths

File: app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py [hunk 2]
```diff
@@ -58,12 +59,22 @@ def tps_addon(pytestconfig, tmpdir_factory):
...
+    resourcesDir = os.path.join(os.getcwd(), resources)
+
+    with open (os.path.join(resourcesDir, 'email.txt'), "w") as f:
         f.write(fxa_account.email)
 
-    with open ("/Users/synctesting/.jenkins/workspace/fenix/app/src/androidTest/resources/password.txt", "w") as f:
+    with open (os.path.join(resourcesDir, 'password.txt'), "w") as f:
         f.write(fxa_account.password)
 
+    os.chdir('../')
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py]: same here

File: app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_integration.py [hunk 3]
```diff
@@ -1,31 +1,27 @@
 import os
 import sys
 
+
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_integration.py]: nit: blank

File: app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_integration.py [hunk 4]
```diff
@@ -1,31 +1,27 @@
...
     gradlewbuild.test('checkBookmarkFromDeviceTest')
-    tps.run('app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_bookmark_desktop.js')
+    tps.run('test_bookmark_desktop.js')
 
 def test_sync_history_from_device(tps, gradlewbuild):
-    os.chdir('app/src/androidTest/java/org/mozilla/fenix/syncintegration/')
     gradlewbuild.test('checkHistoryFromDeviceTest')
-    tps.run('app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_history_desktop.js')
+    tps.run('test_history_desktop.js')
 '''
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_integration.py]: nit: new line

Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_integration.py]: nit: new line

File: app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py (hunk 1 above)
Comment (isabelrios) [reply app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py]: The problem here is that I need to go up in a directory. Form `~/fenix/app/src/androidTest/java/org/mozilla/fenix/syncintegration/` to `~/fenix/app/src/androidTest/` 5 levels up. I can't do that with absolute paths as far as I know. We don't know the beginning of the full path (that will depend on each one's path used) but we do know the end of the path.

**Pull request merged**